http/oidc/src/main/java/org/wildfly/security/http/oidc/ElytronMessages.java

```diff
@@ -278,5 +278,11 @@ interface ElytronMessages extends BasicLogger {
 
     @Message(id = 23070, value = "Authentication request format must be one of the following: oauth2, request, request_uri.")
     RuntimeException invalidAuthenticationRequestFormat();
+
+    @Message(id = 23071, value = "Invalid logout output: %s is not a valid value for %s")
```

Comment:
Now that the OIDC nonce PR was merged, it's a good time to add a commit that merges the latest changes from 2.x and then update the message IDs.

Reply:
done